### DISCUSSION TITLE
How to disable Infolist

### DISCUSSION THREAD

**Discussion by sk-akiyama** [google/mozc#942] [Q&A]
Thank you for always letting me use Google Japanese Input.

I have a question: Is there a way to disable the "usage examples" (Infolist?) shown in the image?
![2024-06-06_16h47_37](https://github.com/google/mozc/assets/48663185/1e0caef7-36f8-47a4-ac48-257efdc2e0cc)

If any adjustments to the source code are necessary, I would appreciate it if you could point out the relevant parts!

Thanks

**Comment by yukawa** [comment]
For this kind of question, please use [discussion](https://github.com/google/mozc/discussions) rather than filing it as an issue. Thanks!

**Comment by yukawa** [comment]
> I have a question: Is there a way to disable the "usage examples" (Infolist?) shown in the image?

Open the config dialog then uncheck `Homonym dictionary` in the `Dictionary` tab.
<img width="396" alt="issue942-en" src="https://github.com/google/mozc/assets/5861397/197c4a38-9b6f-416b-848b-0e1a4ecb9111">

**Comment by sk-akiyama** [comment]
@yukawa 
I apologize for going to the wrong place for the discussion!
And thank you very much!
I should have checked the properties more carefully...
I really appreciate your help.

Have a great day!
